kitsune/questions/views.py

```diff
@@ -468,7 +467,9 @@ def edit_details(request, question_id):
 def aaq_react(request):
     request.session['in-aaq'] = True
     to_json = JSONRenderer().render
-    products = ProductSerializer(Product.objects.filter(visible=True), many=True)
+    products = ProductSerializer(
+        Product.objects.filter(questions_locales__locale=request.LANGUAGE_CODE),
+        many=True)
```

Comment:
I don't understand this change. It went from visible products to products where the locale has AAQ enabled.

Why not visible *and* products where the locale has AAQ enabled?